pkg/sql/logictest/testdata/logic_test/inverted_filter_geospatial_explain_local

```diff
@@ -27,39 +27,35 @@ https://cockroachdb.github.io/distsqlplan/decode.html#eJyUk99v2jAQx9_3V1j3UpC8Yj
+·                          table            geo_table2@geom_index
+·                          spans            31 spans
 
 query TTT
 EXPLAIN SELECT k, k_plus_one FROM geo_table2 WHERE ST_DFullyWithin('POINT(3.0 3.0)'::geometry, geom, 1)
 ----
-·                                    distribution     local
-·                                    vectorized       false
-project                              ·                ·
- └── filter                          ·                ·
-      │                              filter           st_dfullywithin('010100000000000000000008400000000000000840', geom, 1.0)
-      └── index join                 ·                ·
-           │                         table            geo_table2@primary
-           │                         key columns      k, k_plus_one
-           └── project               ·                ·
-                └── inverted filter  ·                ·
-                     │               inverted column  geom_inverted_key
-                     │               num spans        20
-                     └── scan        ·                ·
-·                                    missing stats    ·
-·                                    table            geo_table2@geom_index
-·                                    spans            /1152921504606846976-/1152921504606846977 /1152921521786716160-/1152921521786716161 /1152921526081683456-/1152921526081683457 /1152921527155425280-/1152921527155425281 /1152921527155425281-/1152921527692296191/PrefixEnd /1152921538966585345-/1152921573326323711/PrefixEnd /1152921573326323712-/1152921573326323713 /1152921573326323713-/1152921607686062079/PrefixEnd /1152921607686062081-/1152921642045800447/PrefixEnd /1152921779484753920-/1152921779484753921 /1152922604118474752-/1152922604118474753 /1152925902653358080-/1152925902653358081 /1152939096792891392-/1152939096792891393 /1152991873351024640-/1152991873351024641 /1153202979583557632-/1153202979583557633 /1154047404513689600-/1154047404513689601 /1157425104234217472-/1157425104234217473 /1170935903116328960-/1170935903116328961 /1224979098644774912-/1224979098644774913 /1441151880758558720-/1441151880758558721
+·                          distribution     local
+·                          vectorized       false
+filter                     ·                ·
+ │                         filter           st_dfullywithin('010100000000000000000008400000000000000840', geom, 1.0)
+ └── index join            ·                ·
+      │                    table            geo_table2@primary
+      │                    key columns      k, k_plus_one
+      └── inverted filter  ·                ·
+           │               inverted column  geom_inverted_key
+           │               num spans        20
+           └── scan        ·                ·
+·                          missing stats    ·
+·                          table            geo_table2@geom_index
+·                          spans            20 spans
```

Comment:
shouldn't this scan show some spans and then ... some more? Or is it because it is an inverted index?